### PR TITLE
Search bar airbnb

### PR DESCRIPTION
**Pull request by Oleh-Khashchevskyi**
- [DEMO LINK](https://Oleh-Khashchevskyi.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Oleh-Khashchevskyi.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  border-radius: 3px;
+}
+
+.bigSearch {
+  margin: 20px 0;
+}
+
+.bigSearch__form,
+.smallSearch__form {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```
Comment (illiarerun) [inline src/style.css:53]:
```suggestion
  background: linear-gradient(180deg, #ffffff 0%, #f6f6f7 100%);
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: italic;
+  font-weight: normal;
+}
+
+body {
+  box-sizing: border-box;
+}
```
Comment (illiarerun) [inline src/style.css:25]: It'd be better to set same box-sizing for all elements
```suggestion
* {
  box-sizing: border-box;
}
```

File: src/index.html [hunk 3]
```diff
@@ -4,13 +4,42 @@
...
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="bigSearch">
+      <form
+        action="/"
```
Comment (illiarerun) [inline src/index.html:17]: When i press enter I get the 404 page, fix action
image

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+.search__input {
+  width: 100%;
+  outline: none;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
```
Comment (illiarerun) [inline src/style.css:31]: Redundant font-weight property, remove it
```suggestion
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  background-size: 19px;
+}
+
+.smallSearch__input {
+  padding: 10px 20px 11px 33px;
+  font-size: 14px;
+  font-weight: 400;
```
Comment (illiarerun) [inline src/style.css:69]: Input text should have 900 weight, same for big search

Comment (illiarerun) [reply src/style.css:69]: But placeholder should have normal weight

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+  font-style: normal;
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: italic;
+  font-weight: normal;
+}
```
Comment (illiarerun) [inline src/style.css:21]: You are not using these fonts, delete them, or find a usage for them
```suggestion
```

**Comment by Oleh-Khashchevskyi** [comment]
Everything is OK

**Comment by illiarerun** [comment]
Yeah, now tests are OK, can you reopen your PR so I can review it?)

**Pull request closed without merging**